html.cfg

```diff
@@ -51,9 +51,21 @@
     right: 50px;
 }}
 
-\begin{document}
-\EndPreamble
+\Css{.right {
+    text-align: right;
+}}
 
-\begin{flushright}
-\href{https://github.com/sysprog21/lkmpg/releases/download/latest/lkmpg.pdf}{Download PDF document}
-\end{flushright}
\ No newline at end of file
+\AtBeginDocument{%
+\Configure{@HEAD}{\HCode{
+    <script async defer src="https://buttons.github.io/buttons.js"></script>
+    <div class="right">
+        <a class="github-button" href="https://github.com/sysprog21/lkmpg" data-size="large" aria-label="Check the repo on GitHub">Check the repo on GitHub</a>
+        <br><br>
+        <a href="https://github.com/sysprog21/lkmpg/releases/download/latest/lkmpg.pdf">Download PDF document</a>
```

Comment:
Is it possible to use the similar button to download PDF? https://buttons.github.io/

Reply:
Yes, however it cause different width between 2 buttons:
![image](https://user-images.githubusercontent.com/58484289/131526773-afb769f8-280a-4dd5-b6a5-8af94f8e320e.png)
And currently author don't provide a way to resize the button.[1]

To align them, we need to assure they have same number of characters.
Or maybe put them in the same line:
![image](https://user-images.githubusercontent.com/58484289/131528682-8880ade8-4564-468d-a3e2-c78058f6e7e4.png)

[1] - https://github.com/ntkme/github-buttons/issues/34

Reply:
I think it makes sense to put two buttons on the same line.